txpipe/plotting/correlations.py

```diff
@@ -352,7 +353,13 @@ def make_theory_plot_data(data, cosmo, obs, label, smooth=True, xi=None):
             # Optionally also compute correlation functions
             if xi:
                 theta, *_ = obs[(GAMMA, i, j)]
-                theory[GAMMA, i, j] = theta, pyccl.correlation(cosmo, ell, cl, theta/60, corr_type='GL')
```

Comment:
I fixed this in CCL.

Reply:
Great! so then this means I should remove this if statement I added right?